src/event-stream/event-server.ts

```diff
@@ -214,21 +221,60 @@ async function handleClientMessage(msg: CoreNodeBlockMessage, db: DataStore): Pr
           if (nameFunctions.includes(functionName)) {
             const attachment = parseNameRawValue(event.contract_event.raw_value);
             const attachmentValue = await fetchAttachmentContent(attachment.attachment.hash);
-            const name: DbBNSName = {
-              name: attachment.attachment.metadata.name,
-              namespace_id: attachment.attachment.metadata.namespace,
-              address: addressToString(attachment.attachment.metadata.tx_sender),
-              expire_block: 0, // FIXME:
-              registered_at: parsedMsg.block_height,
-              zonefile_hash: attachment.attachment.hash,
-              zonefile: attachmentValue,
-              latest: true,
-              tx_id: event.txid,
-              status: attachment.attachment.metadata.op,
-              index_block_hash: parsedMsg.index_block_hash,
-              canonical: true,
-            };
-            dbTx.names.push(name);
+            let caseForSubdomain = false;
```

Comment:
Quite a bit of new code in the event handler that isn't covered by any tests. We can merge into the bns branch, but it we will need tests covering these before merging into master.